### ISSUE TITLE
[Idea] Improvements for users profile

### ISSUE THREAD

**Issue by ghost** [larastream/internals#56]
Because Larastream is for Laravel, I think users should get a better chance to present their projects or websites instead of to use the little description field. Maybe it's just a vision but on this way, they also could generate some visitor for their things.

When users also using social medias like Facebook, Twitter, Github, Bitbucket or something else and they want to illustrate their profiles then they should have the opportunity to set up their profiles in Larastream.

**Comment by JordanDalton** [comment]
I agree. I'll see about coming up with a concept (feel free to share one if you have it).

**Comment by vmitchell85** [comment]
I like what some sites do and provide specific fields for twitter, facebook, github etc...
